test/evergreen.yml

```diff
@@ -2546,7 +2546,7 @@ tasks:
           script: |
             set -o errexit
             set -o verbose
-            sh s_all -A -E 2>&1
+            sh -x s_all -A -E 2>&1
```

Comment:
I deliberately left this in. With this change we have a chance of debugging and diagnosing issues with s_all via Evergreen.